### PR TITLE
Fixed a bunch of typos in the docs

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Comment by alex** [comment]
Tests are failing, but at least the first one I spotted doesn't look related:

```
../_srcdist/apps/asn1pars.c:105:5: error: no previous prototype for ‘asn1parse_main’ [-Werror=missing-prototypes]
```

**Comment by richsalz** [comment]
Build is broken right now, not your fault ☺  Will be fixed soon.

File: doc/apps/ca.pod [hunk 1]
```diff
@@ -101,7 +101,7 @@ section for information on the required input and output format.
 =item B<-infiles>
 
 if present this should be the last option, all subsequent arguments
-are assumed to the the names of files containing certificate requests. 
+are assumed to the names of files containing certificate requests.
```
Comment (richsalz) [inline doc/apps/ca.pod]: should be "assumed to be the"  or perhaps better "are taken as"

Comment (alex) [inline doc/apps/ca.pod]: Fixed.

**Comment by richsalz** [comment]
perhaps we can factor out the -rand thing into "see the -rand flag in openssl"?  We already have "PASS PHRASE ARGUMENTS" in openssl.pod. so moving more common flags there seems to make sense.

**Comment by richsalz** [comment]
like make "PASS PHRASE ARGUMENTS" be a head2, and put a new head1 COMMON ARGUMENTS before it, to "wrap" around that?  does this make sense?

**Comment by alex** [comment]
While that sounds completely reasonable, I'd prefer to expand the scope of this PR. It seems like this could be merged as-is and a separate ticket filed to do that simplification?

**Comment by levitte** [comment]
+1

and I agree with @alex, take other doc issues in separate PR.

**Comment by richsalz** [comment]
Sure. that's fine.

**Comment by richsalz** [comment]
okay, pushed to master.  thanks!

**Pull request closed without merging**